.ci/lib.sh

```diff
@@ -281,6 +291,13 @@ gen_clean_arch() {
 		GOCACHE=${GOCACHE:-$HOME/.cache/go-build}
 	fi
 	[ -d "$GOCACHE" ] && sudo rm -rf ${GOCACHE}/*
+
+	info "Clean stale lock files"
+	# stale lock files are created when processes are not terminated properly
+	stale_lock_file_union=( "/var/lib/dpkg/lock" )
```

Comment:
It's slightly odd having this global var set here, but used in the other func. I'd probably either define it in the other fun, or pass it through as an argument?